### ISSUE TITLE
template-path command line argument does not work

### ISSUE THREAD

**Issue by bmuenzenmeyer** [transitive-bullshit/create-react-library#93]
👋 super cool library, thanks!

When I attempt to use CRL with the `--template-path` argument, the specified value is not supplied to inquirer as a default, nor is it supplied if `--skip-prompts` is supplied.

To reproduce:

```
create-react-library component-test --desc Reproduction --author bmuenzenmeyer --no -git --license UNLICENSED --manager npm --repo component-test --template custom --template-path ./component-template
```

![image](https://user-images.githubusercontent.com/298435/47528457-dd8df080-d86a-11e8-9ebd-8d28169a0774.png)

When `--skip-prompts` is specified, a path error is thrown

![image](https://user-images.githubusercontent.com/298435/47528509-feeedc80-d86a-11e8-9a7c-e11f016d0732.png)

To fix, `templatePath` needs to be fed into `promptLibraryParams`

**index.js**

``` diff
  const opts = {
    description: program.desc,
    author: program.author,
    license: program.license,
    repo: program.repo,
    manager: program.manager,
    template: program.template,
+   templatePath: program.templatePath,
    skipPrompts: program.skipPrompts,
    git: program.git,
  }
```

and defaulted within **prompt-library-params.js**

``` diff
...
      {
        type: 'input',
        name: 'templatePath',
        message: 'Template Path',
        when: ({ template }) => template === 'custom',
+       default: opts.templatePath,
        validate: input =>
          new Promise(resolve => {
            const fullPath = path.resolve(process.cwd(), input)
            fs.stat(fullPath, (err, stats) => {
              if (err) {
                return resolve(`Cannot resolve directory at: ${fullPath}`)
              }
              resolve(true)
            })
          }),
      },
...
```

With monkey patch, the specified value is honored as a default. `--skip-prompts` then works too:

![image](https://user-images.githubusercontent.com/298435/47528772-aa982c80-d86b-11e8-91a7-32754e5d0c4d.png)

I will submit a PR for this. Hoping to add coverage of this scenario too, if time allows and I can grok the test suite.

**Comment by zachwolf** [comment]
Semi related: it seems like a mistake that options passed as flags still produce prompts.

If I have a npm script that looks like `"create-component": "create-react-library --manager npm"`, I would expect running `npm run create-component` to skip `? Package Manager npm`. Thoughts?

**Comment by transitive-bullshit** [comment]
Fix released as `v2.6.7`.
